server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/LoginPage.java

```diff
@@ -41,26 +39,26 @@ public Navigation submitCredentials(String login, String password) {
   }
 
   public Navigation useOAuth2() {
-    $(".oauth-providers a").click();
-    return page(Navigation.class);
+    Selenide.$(".oauth-providers a").click();
+    return Selenide.page(Navigation.class);
   }
 
   public LoginPage submitWrongCredentials(String login, String password) {
-    $("#login").val(login);
-    $("#password").val(password);
-    $(By.name("commit")).click();
-    return page(LoginPage.class);
+    Selenide.$("#login").val(login);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "#login" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)